$MirOS: ports/devel/automake/1.4/patches/patch-automake_in,v 1.1.7.1 2005/03/18 15:44:28 tg Exp $
$OpenBSD: patch-automake_in,v 1.8 2001/08/27 14:01:47 espie Exp $
--- automake.in.orig	Sat Jul 27 20:55:36 2002
+++ automake.in	Mon Mar  6 23:06:23 2006
@@ -30,6 +30,10 @@ eval 'exec @PERL@ -S $0 ${1+"$@"}'
 # Parameters set by configure.  Not to be changed.  NOTE: assign
 # VERSION as string so that eg version 0.30 will print correctly.
 $VERSION = "@VERSION@";
+$MPVERSION = '@MPVERSION@';
+if ($MPVERSION eq '@MPVER'.'SION@') {
+  $MPVERSION = $VERSION;
+}
 $PACKAGE = "@PACKAGE@";
 $prefix = "@prefix@";
 $am_dir = "@datadir@/@PACKAGE@-@APIVERSION@";
@@ -2012,7 +2016,7 @@ sub handle_texinfo
 	# dependency list.
 	@texi_deps = ();
 	push (@texi_deps, $info_cursor);
-	push (@texi_deps, $vtexi) if $vtexi;
+	push (@texi_deps, '$(srcdir)/'.$vtexi) if $vtexi;
 
 	# Canonicalize name first.
 	($canonical = $infobase) =~ tr/A-Za-z0-9_/_/c;
@@ -3545,7 +3549,7 @@ sub handle_merge_targets
     &do_one_merge_target ('install-data', @install_data);
     $output_rules .= "\n";
 
-    &do_one_merge_target ('install', 'all-am');
+    &do_one_merge_target ('install');
     &do_one_merge_target ('uninstall', @uninstall);
 
     &do_one_merge_target ('all', @all);
@@ -6163,7 +6167,7 @@ sub read_main_am_file
     local ($amfile) = @_;
 
     $output_vars = ("# $in_file_name generated automatically by automake "
-		    . $VERSION . " from $am_file_name\n");
+		    . $MPVERSION . " from $am_file_name\n");
 
     # Generate copyright for generated Makefile.in.
     $output_vars .= $gen_copyright;
